### PR TITLE
Refactor/consts

### PR DESCRIPTION
**Pull request by michalirak**
Moved all consts to new file, moved it into the main project

### CODE REVIEW

File: server/discord.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+from typing import Generator, List
 
-from selenium.common.exceptions import StaleElementReferenceException, NoSuchElementException
+from selenium.common.exceptions import StaleElementReferenceException
 from selenium.webdriver.chrome.webdriver import WebDriver
 from selenium.webdriver.common.by import By
 from selenium.webdriver.remote.webelement import WebElement
 
-from consts import DISCORD_LOGIN_URL, SERVER_CLASS_NAME
+from server.consts import *
```
Comment (michalirak) [inline server/discord.py:9]: Not sure about this line
I could write it all out explicitly, but also we would definitely want to use all the available exceptions.
What do you think?

File: server/discord.py [hunk 2]
```diff
@@ -66,9 +65,9 @@ def _get_unread_messages(self, last_message: str, one_before_message: str,  mess
         return unread_messages[::-1]
 
     def _get_in_chat(self, chat_name: str):
-        chats = self._browser.find_elements(By.CLASS_NAME, "containerDefault--pIXnN")
+        chats = self._browser.find_elements(By.CLASS_NAME, CHANNEL_CLASS)
         for chat in chats:
-            if chat.get_attribute("data-dnd-name") == chat_name:
+            if chat.get_attribute(CHANNEL_NAME_ATTRIBUTE) == chat_name:
                 chat.click()
                 sleep(3)
```
Comment (michalirak) [inline server/discord.py:94]: Why?

File: server/consts.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+DISCORD_LOGIN_URL = "https://discord.com/login"
+
+# HTML CONSTS
+SERVERS_DIV_XPATH = "//div[@aria-label='Servers']"
```
Comment (michalirak) [inline server/consts.py:4]: Isn't this a much much better xpath? :)

File: server/discord.py (hunk 2 above)
Comment (shahar481) [reply server/discord.py:94]: Hack because it takes time to load

**Pull request merged**